### PR TITLE
Warn that the Debian package is outdated

### LINKED ISSUE #213: Haskell platform on Debian is sadly out of date
The Haskell Platform for Debian Linux asks users to use APT to install the Haskell Platform.  But the recently released Debian stable distribution (code name jessie) includes a Haskell Platform from 2013.  It would be fantastic to provide a current  Haskell Platform that is compiled against jessie, and who knows, perhaps the Debian people would include in in backports?  But if that doesn't work, there should be another path for people who need or want a current Haskell Platform.

### PR DESCRIPTION
**Pull request by neongreen**
See https://www.reddit.com/r/haskell/comments/43wzxn/how_do_i_get_current_version_of_haskell/

Fixes #213.

### CODE REVIEW

**Comment by 23Skidoo** [comment]
Merged, thanks!

**Comment by gbaz** [comment]
Actually I'll tweak the language. I want to add "If you want the latest, you are advised..." -- some people really do want to use what comes with their distro even if its not the most recent. That's why we provide the info!

**Comment by gbaz** [comment]
I changed things to this effect and also pulled the warning to the top level, as the out-of-date issue affects all distro-packaged versions:

https://github.com/haskell/haskell-platform/commit/fa4286aa4bcf296474f89f8d0a2b5aa7f914c888

**Comment by neongreen** [comment]
@gbaz: sure, but “if you prefer to use the latest” suggests that using what comes with one's distro is the default, while “latest” is for people who know what they are doing / who have some special preference. In reality, the opposite is probably true (at least for distros like Debian) – the default should be GHC 7.10, while GHC 7.6 (which is shipped with HP 2013) should be for people who know what they're doing (i.e. “I'm fine with GHC 7.6, using the distro package is more important for me”).

In other words, my point is that Haskell (well, GHC Haskell) is a fast-moving language, and I think we ought to warn beginners that using 2-year-old GHC could be more unpleasant than they think it would be.

**Comment by gbaz** [comment]
Personally I think what people may or may not find pleasant vis a vis ghc releases really will vary by their use case. 

I'm open to any language suggestions that make things more _neutral_ but don't want to steer people in either direction.

Core package maintainers are encouraged to support a three-major-release backwards support window. GHC 7.6 falls outside that, I'll grant. Perhaps we can amend the text to note that three-major-release window as a way to judge...
